samples/bluetooth/peripheral_uart/src/main.c

```diff
@@ -32,15 +33,15 @@
 
 /* Change this if you have an LED connected to a custom port */
 #ifndef LED0_GPIO_CONTROLLER
-#define LED0_GPIO_CONTROLLER    LED0_GPIO_PORT
+#define LED0_GPIO_CONTROLLER    DT_GPIO_LEDS_LED_0_GPIO_CONTROLLER
```

Comment:
I am not sure which name should be used. Maybe @anangl will know something more.

Reply:
The deprecation policy of these non-DT_ macros in not quite clear to me, but apparently these short macros generated for LEDs and buttons are not deprecated, so it seems to be perfectly OK to use them in applications. And since they are shorter and nicer-looking, I support @lemrey's suggestion.

Reply:
Macro changed to short.